x-pack/plugins/enterprise_search/public/applications/app_search/components/layout/kibana_header_actions.test.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockValues } from '../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiButtonEmpty } from '@elastic/eui';
+
+import { KibanaHeaderActions } from './kibana_header_actions';
+
+describe('KibanaHeaderActions', () => {
+  const values = {
+    engineName: 'foo',
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<KibanaHeaderActions />);
+    expect(wrapper.find(EuiButtonEmpty).exists()).toBe(true);
+  });
+
+  it('does not render a "Query Tester" button if there is no engine available', () => {
```

Comment:
This test will change in the next PR. I'll most likely have a "QueryTesterButton" or something more specific to look for.